clients/src/main/java/org/apache/kafka/common/Uuid.java

```diff
@@ -49,21 +49,21 @@ public class Uuid implements Comparable<Uuid> {
     /**
      * A UUID that is used to identify new or unknown dir assignments.
      */
-    public static final Uuid UNKNOWN_DIR = ZERO_UUID;
+    public static final Uuid UNASSIGNED_DIR = ZERO_UUID;
```

Comment:
The `Uuid` class should not know about any of this. We should move this logic to the module where it makes sense.

Reply:
Ok. Directory IDs will be used from metadata, core, storage and tools. I've moved them to a separate class, but I'm not sure which module would be a better fit than clients for this. Do you have a suggestion?

Reply:
`server-common` probably.

Reply:
Thanks for the suggestion. I've moved the class to that module.